Competition2017/src/competition/subsystems/drive/commands/StopDriveCommand.java

```diff
@@ -0,0 +1,25 @@
+package competition.subsystems.drive.commands;
+
+import competition.subsystems.drive.DriveSubsystem;
+
+public class StopDriveCommand extends BaseDriveCommand {
+
+    public StopDriveCommand(DriveSubsystem driveSubsystem) {
+        super(driveSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        log.info("initializing");
+    }
+
+    @Override
+    public void execute() {
+        driveSubsystem.stop();
```

Comment:
Can we put this in `initialize` as well to ensure that it runs if `execute` isn't called before `isFinished`?